src/core/p5.Renderer2D.js

```diff
@@ -241,7 +242,12 @@ p5.Renderer2D.prototype.get = function(x, y, w, h) {
     var sw = dw * pd;
     var sh = dh * pd;
 
-    var region = new p5.Image(dw, dh);
+    var region;
+    if(this.cachedGetImage === null) {
```

Comment:
ah yeah oops! I just fixed it so that it does but now `b` is modified along with `c` in the above example